### PR TITLE
Fix a bug where sf-sns-report didn't return the right payload + more integration tests

### PR DESCRIPTION
**Pull request by scisco**
**Summary:** Summary of changes

Addresses [CUMULUS-441](https://bugs.earthdata.nasa.gov/browse/CUMULUS-441)

## Changes

* Fixed a bug where the sf-sns-report task did not return the payload of the incoming message as the output of the task
* Use sf-sns-report in all integration test workflows
* Fix a bug in the integration tests where discover/queue pdrs was intermittently failed

## Test Plan
Things that should succeed before merging.

- [x] Unit tests
- [x] Adhoc testing
- [x] Update CHANGELOG
- [x] Run `./bin/eslint-ratchet` and verify that eslint errors have not increased
  - [x] Commit `.eslint-ratchet-high-water-mark` if the score has improved

### CODE REVIEW

File: example/spec/discoverAndQueuePdrs/DiscoverAndQueuePdrsSuccessSpec.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ describe('The Discover And Queue PDRs workflow', () => {
   let workflowExecution = null;
 
   beforeAll(async () => {
+    await deleteFolder(config.bucket, `${config.stackName}/pdrs`);
+
```
Comment (abarciauskas-bgse) [inline example/spec/discoverAndQueuePdrs/DiscoverAndQueuePdrsSuccessSpec.js]: Should we remove the `afterAll` block?

Comment (scisco) [reply example/spec/discoverAndQueuePdrs/DiscoverAndQueuePdrsSuccessSpec.js]: yeah, I think we could. will remove it.

**Comment by scisco** [comment]
thanks @jennyhliu. It will replace these when we merge.

**Comment by scisco** [comment]
The full aws integration test also passed here: https://circleci.com/gh/cumulus-nasa/cumulus/1994

**Pull request merged**